tests/unit/replication/ReplicationQueuePopulator.spec.js

```diff
@@ -149,4 +149,68 @@ describe('replication queue populator', () => {
         // should not throw
         rqp._filterBucketOp(entry);
     });
+
+    // A "standalone null master key" is created when an object is placed in a non-versioned bucket,
+    // which is then converted to a versioned bucket. If no new versioned objects are added for that object,
+    // it appears as a standalone null master key with no version id.
+    it('should replicate standalone null master key', () => {
+        const customKafkaValiue = {
```

Comment:
Here and in other tests where this typo can be found:

```suggestion
        const customKafkaValue = {
```